docs/src/api.md

```diff
@@ -36,25 +36,53 @@ isoper
 isoperbra
 isoperket
 issuper
+LinearAlgebra.ishermitian
+isherm
+LinearAlgebra.issymmetric
+LinearAlgebra.isposdef
 ```
 
 ## [Qobj arithmetic and attributes](@id doc-API:Qobj-arithmetic-and-attributes)
 
 ```@docs
-sqrtm
-sinm
-cosm
+Base.conj
+LinearAlgebra.transpose
+LinearAlgebra.adjoint
+LinearAlgebra.dot
+LinearAlgebra.sqrt
+LinearAlgebra.log
+LinearAlgebra.exp
 LinearAlgebra.tr
 LinearAlgebra.svdvals
 LinearAlgebra.norm
+LinearAlgebra.normalize
+LinearAlgebra.normalize!
+LinearAlgebra.inv
+LinearAlgebra.diag
+sinm
```

Comment:
Can you convert also `sinm` and `cosm to `LinearAlgebra.sin` and `LinearAlgebra.cos` and them put `sinm` and `cosm` as synonyms?

Reply:
Ohh, I didn't notice that there is also matrix `sin` and `cos` in `LinearAlgebra`.
Will implement it right away !

Reply:
This could be more efficient compared to the original way

Reply:
No I don’t think that it will work with the LinearAlgebra functions, because we have sparse matrices, and (at least at the time) there wasn’t an implementation for such functions. That’s why I implemented the `exp` function by myself. When SparseArrays will export their `exp` `sin` `cos` functions, we can delete our definition of `exp` function.

What I mean here is to do the same as the `exp` case also for `sin` and `cos`, and then define `sinm` and `cosm` as synonyms.

Reply:
Okay, just to make sure I understand correctly, you mean:
```julia
LinearAlgebra.sin(A::QuantumObject) = (exp(1im * A) - exp(-1im * A)) / 2im
sinm(A::QuantumObject) = sin(A)
```